File: cellulosesz-api/src/main/java/top/likoslupus/cellulosesz/api/item/WorkstationKind.java
package top.likoslupus.cellulosesz.api.item;

import java.util.Locale;

public enum WorkstationKind {

    ANVIL,
    CARTOGRAPHY,
    DISPOSAL,
    GRINDSTONE,
    LOOM,
    SMITHING,
    STONECUTTER,
    WORKBENCH;

    public String permissionSegment() {
        return name().toLowerCase(Locale.ROOT);
    }

}

File: cellulosesz-api/src/main/kotlin/top/likoslupus/cellulosesz/api/item/WorkstationKind.kt
package top.likoslupus.cellulosesz.api.item

import java.util.*

public enum class WorkstationKind {

    ANVIL,
    CARTOGRAPHY,
    DISPOSAL,
    GRINDSTONE,
    LOOM,
    SMITHING,
    STONECUTTER,
    WORKBENCH;

    public fun permissionSegment(): String = name.lowercase(Locale.ROOT)

}
